test-support/src/main/java/org/springframework/ldap/test/EmbeddedLdapServerFactoryBean.java

```diff
@@ -20,6 +20,7 @@
 
 /**
  * @author Mattias Hellborg Arthursson
+ * @author Eddu Melendez
```

Comment:
You probably don't need your name here, but you can place it on the test.apacheds2 version